htdocs/admin/ldap_users.php

```diff
@@ -184,7 +184,7 @@
 print '<tr class="oddeven"><td>'.$langs->trans("LDAPFieldFirstName").'</td><td>';
 print '<input size="25" type="text" name="fieldfirstname" value="'.$conf->global->LDAP_FIELD_FIRSTNAME.'">';
 print '</td><td>'.$langs->trans("LDAPFieldFirstNameExample").'</td>';
-print '<td align="right"><input type="radio" name="key" value="LDAP_FIELD_FIRSTNAME"'.(($conf->global->LDAP_KEY_USERS && $conf->global->LDAP_KEY_USERS==$conf->global->LDAP_FIELD_FIRSTNAME)?' checked':'')."></td>";
+print '<td align="right">&nbsp;</td>';
```

Comment:
If selection of the key that is the common key to make sync is removed, how can we know what is the "common" key between the 2 systems ?

Reply:
@eldy 
Do we agree that the ldap key is used to create the DN of the record? In general it is often the "first name / name", "uid", "email" ... I have restored some fields. I will also test later the function "ldap_rename" which is compatible only with version 3 of the protocol and which would not lose the password.

http://www.commentcamarche.net/contents/635-ldap-le-modele-de-nommage

http://php.net/manual/fr/function.ldap-rename.php